dev/dags/.airflowignore.old

```diff
@@ -0,0 +1,12 @@
+basic_cosmos_dag.py
```

Comment:
Do we need this file? If yes, would it make sense to add a comment at the top of the file mentioning the purpose of this file.

Reply:
No, just deleted: 3cb6bedefd79c315e7308f2befdbb6343b4d81ca